taskRuby.rb

```diff
@@ -0,0 +1,23 @@
+#Functions
+def ask(question)
+    print question
+    end
+    def Hrs(hr)
+    hr*365*24

```

Comment:
I would probably name this argument `years` instead of `hrs` because thats actually what you are taking in.  You are outputing a number of hours